llms/mlx_lm/server.py

```diff
@@ -159,6 +161,40 @@ def do_POST(self):
         method = self.handle_stream if self.stream else self.handle_completion
         method(prompt, stop_id_sequences)
 
+    def validate_model_parameters(self):
+        """
+        Validate the model parameters passed in the request for the correct
+            types and values
```

Comment:
```suggestion
        Validate the model parameters passed in the request for the correct
        types and values
```

Reply:
I had the tab to be consistent with other docstrings in the same file. For example, https://github.com/ml-explore/mlx-examples/blob/main/llms/mlx_lm/server.py#L171

Reply:
I think the style we usually use in MLX is that this should be 1 line and end in a period. The others should be fixed as well.